clang/docs/ReleaseNotes.rst

```diff
@@ -352,6 +352,10 @@ Improvements to Clang's diagnostics
 - Now correctly diagnose a tentative definition of an array with static
   storage duration in pedantic mode in C. (#GH50661)
 
+- Split diagnosis of implicit integer comparison on negation to a new
+  diagnostic group ``-Wimplicit-int-comparison-on-negation``, grouped under
```

Comment:
typo here: it spells `-Wimplicit-int-conversion-on-negation` in the actual implementation